src/main/asciidoc/reference-guide/tutorials/configdatabaseresource.adoc

```diff
@@ -0,0 +1,160 @@
+=== Configure a Database resource
+:hardbreaks:
```

Comment:
There is no need to add such index: AsciiDoctor generates it automatically.
Please remove it.

Reply:
Which line are you referring to?
Without ":hardbreaks:", sentences on different lines are being collapsed together into a single line

Reply:
Sorry, I was meaning the lines from 2 to 11, which start with `:hardbreaks:`, until before `==== Introduction`